packages/frontend/src/App.tsx

```diff
@@ -244,6 +245,7 @@ const App = () => (
                                         </div>
                                     </PrivateRoute>
                                 </Switch>
+                                <RefreshStepsModal />
```

Comment:
@owlas @rephus I have moved it to the app, so the modal can be open from everywhere in the app

Reply:
Nice!!